src/MPDevice.cpp

```diff
@@ -1994,7 +1994,7 @@ bool MPDevice::addChildToDB(MPNode* parentNodePt, MPNode* childNodePt)
         else
         {
             /* If the login name is supposed to be before the one we're trying to insert */
-            if (tempNextChildNodePt->getLogin() > childNodePt->getLogin())
+            if (tempNextChildNodePt->getLogin().compare(childNodePt->getLogin()) > 0)
```

Comment:
here's the code line: https://github.com/limpkin/mooltipass/blob/master/source_code/src/NODEMGMT/node_mgmt.c#L1195